TAGradingServer/account/admin-gradeable.php

```diff
@@ -308,21 +312,30 @@ function selectBox($question, $grade = 0) {
             style="cursor: auto; background-color: #FFF; width: 250px;">
             <br />
 
-
-       <!-- <br />
-        Is this a team assignment?:
-        <input type="radio" name="team_assignment" value="yes"
+            Is this a team assignment?:
+            <fieldset>
+            <input type="radio" id = "team_yes_radio" class="team_yes" name="team_assignment" value="yes"
 HTML;
-    
     echo ($g_team_assignment===true)?'checked':'';
     print <<<HTML
         > Yes
-            <input type="radio" name="team_assignment" value ="no" 
+            <input type="radio" class="team_no" name="team_assignment" value ="no" 
 HTML;
     echo ($g_team_assignment===false)?'checked':'' ;
     print <<<HTML
```

Comment:
Set a variable for these two echo statements and then print them inside the heredoc instead of opening and closing the heredoc in the middle of input tags which is a source of potential confusion.